lib/ansible/modules/cloud/amazon/aws_organizations.py

```diff
@@ -0,0 +1,290 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Kamil Potrec <kamilpotrec@googlemail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+module: aws_organizations
+short_description: Manages AWS Organization
+description:
+  - Creates and deletes AWS OrganizationalUnits
```

Comment:
Nit: please see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block

> Descriptions should always start with a capital letter and end with a full stop. Consistency always helps.

```suggestion
  - Creates and deletes AWS OrganizationalUnits.
```

Reply:
Fixed